integrationtests/src/test/java/io/kroxylicious/proxy/KrpcFilterIT.java

```diff
@@ -113,208 +107,218 @@ private static ByteBuffer decode(String topicName, ByteBuffer in) {
     }
 
     @Test
-    public void shouldPassThroughRecordUnchanged() throws Exception {
+    public void shouldPassThroughRecordUnchanged(KafkaCluster cluster) throws Exception {
         String proxyAddress = "localhost:9192";
+        try (var admin = Admin.create(Map.of(AdminClientConfig.BOOTSTRAP_SERVERS_CONFIG, cluster.getBootstrapServers()))) {
```

Comment:
You can use the declarative injection for the Admin client.  (The producer needs to stay as is, as is needs the proxy address).

Reply:
We can address this another day.

Reply:
I was working on it. Done